packages/wallets/onto/package.json

```
@@ -0,0 +1,38 @@
+{
+    "name": "@solana/wallet-adapter-onto",
+    "version": "0.3.6",
+    "author": "Solana Maintainers <maintainers@solana.foundation>",
+    "repository": "https://github.com/solana-labs/wallet-adapter",
+    "license": "Apache-2.0",
+    "type": "module",
+    "sideEffects": false,
+    "main": "lib/cjs/index.js",
+    "module": "lib/esm/index.js",
+    "types": "lib/esm/index.d.ts",
+    "exports": {
+        "import": "./lib/esm/index.js",
+        "require": "./lib/cjs/index.js"
+    },
+    "files": [
+        "lib",
+        "src",
+        "LICENSE"
+    ],
+    "publishConfig": {
+        "access": "public"
+    },
+    "scripts": {
+        "clean": "shx rm -rf lib/*",
+        "build": "yarn clean && tsc -p tsconfig.json && tsc-esm -p tsconfig.json && tsc -p tsconfig.cjs.json",
+        "postbuild": "echo '{\"type\":\"commonjs\"}' | npx json > lib/cjs/package.json && echo '{\"type\":\"module\"} ' | npx json > lib/esm/package.json"
+    },
+    "peerDependencies": {
+        "@solana/web3.js": "^1.44.3"
+    },
+    "dependencies": {
+        "@solana/wallet-adapter-base": "^0.9.9"
+    },
+    "devDependencies": {
+        "@solana/web3.js": "^1.44.3"
+    }
```

```suggestion
    "peerDependencies": {
        "@solana/web3.js": "^1.50.1"
    },
    "dependencies": {
        "@solana/wallet-adapter-base": "^0.9.12"
    },
    "devDependencies": {
        "@solana/web3.js": "^1.50.1"
    }
```
Just updating these for consistency with the other packages.